audio-worklet/src/bitcrusher-demo.js

```diff
@@ -124,13 +150,13 @@ class BitcrusherDemo {
    */
   switchBitcrusherNode() {
     // The function is called after the checkbox switches state.
-    const changeToScriptProcessor = !this.useAudioWorkletCheckBox_.checked;
-    if (changeToScriptProcessor) {
-      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
-      this.bitcrusherAudioWorklet_.disconnect();
-    } else {
+    const changeToAudioWorklet = !this.useScriptProcessorCheckBox_.checked;
+    if (changeToAudioWorklet) {
       this.bitcrusherAudioWorklet_.connect(this.masterGain_);
       this.bitcrusherScriptProcessor_.output.disconnect();
+    } else {
+      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
+      this.bitcrusherAudioWorklet_.disconnect();
```

Comment:
As I said, you connect before disconnect.  There's a possibility that you'll have both going to the output because connect and disconnect aren't sample accurate.  If you disconnect before connect, there's a chance where nothing is connected causing yet another glitch. Of course, there will be a glitch anyway since the delays aren't the same between the two paths, but no need to introduce more sources of glitching.

Gain nodes with automation will fix this.

And does it really matter there are extra gain nodes?  It shouldn't.